PROTOCOL.md

```diff
@@ -331,6 +347,31 @@ The following is an example `remove` action.
 }
 ```
 
+### Add CDC File
```

Comment:
The protocol version information is not here.

Reply:
Added two subsections in this area to clarify writer/reader requirements as you've asked. Let me know if you'd like to move it elsewhere. Change Data Feed is unlike the other features (like Column Mapping) so there's no clear precedent on how to organize it within this doc.